packages/studio/src/components/PreviewWithUseComponents.tsx

```diff
@@ -3,24 +3,27 @@ import useStudioStore from "../store/useStudioStore";
 import PreviewPanel from "./PreviewPanel";
 import Highlighter from "./Highlighter";
 import IFramePortal from "./IFramePortal";
-import { CSSProperties } from "react";
+import { Tooltip, IPosition } from "react-tooltip";
 import { useState } from "react";
-import { Tooltip } from "react-tooltip";
 
-const inlineStyles: CSSProperties = {
-  overflow: "scroll",
-};
+export interface tooltipProps {
```

Comment:
can we directly use the ITooltip type that's exported from react-tooltip instead of creating our own?

Reply:
Ohhh yes